Makefile

```diff
@@ -14,7 +14,7 @@ migrate:
 superuser:
 	docker compose run --rm web poetry run ./manage.py createsuperuser
 
-site:
+site: migrate
```

Comment:
Not sure what this is doing. Just checking that it was intentional 😄

Reply:
This was.  I am always trying to improve local environment.